modules/op-configuration-options-with-default-values.adoc

```diff
@@ -0,0 +1,60 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_content-type: REFERENCE
+[id="op-configuration-options-with-default-values_{context}"]
+= Configuration options with default values
+
+[role="_abstract"]
+The following is a list of modifiable fields with default values in the `TektonConfig` CR.
+
+* `running-in-environment-with-injected-sidecars` (default: `true`): Set this option to `false` if pipelines are running in a cluster that do not use injected sidecars, such as Istio. Setting it to `false` decreases the time it takes for a task run to start.
++
+NOTE: For clusters that use injected sidecars, setting this option to `false` can lead to unexpected behavior.
+
+* `await-sidecar-readiness` (default: `true`): Set this option to `false` to stop {pipelines-shortname} from waiting for `TaskRun` sidecar containers to be running before it begins to operate. This allows tasks to be run in environments that do not support the `downwardAPI` volume type.
++
+NOTE: For clusters that use injected sidecars, setting this option to `false` can lead to unexpected behavior.
+
+* `default-service-account` (default: `pipeline`): This option contains the default service account name to use for `TaskRun` and `PipelineRun`, if none is specified.
+
+* `require-git-ssh-secret-known-hosts` (default: `false`): Setting this option to `true` requires that any Git SSH secret must include the `known_hosts` field. 
+
+** See _Configuring SSH authentication for Git_ in the _Additional resources_ section for more information about Git SSH secrets.
+
+* `enable-tekton-oci-bundles` (default: `false`): Set this option to `true` to enable the use of an experimental alpha feature, Tekton OCI bundle.
+
+* `send-cloudevents-for-runs` (default: `false`): Setting this option to `true` enables `CloudEvents` for runs and custom runs, if the `CloudEvents` sink is configured in the `config-defaults` configuration map.
```

Comment:
@vdemeester is this something that we support?

Reply:
Define support 😛 it works, we don't really "support" it in the RH sense I think.

Reply:
Thanks for the clarification. As it is not officially supported by Red Hat, I removed the `send-cloudevents-for-runs` field.